tool-plugins/intellij/README.md

```diff
@@ -4,73 +4,35 @@
+    - [Creating a new Ballerina file](https://ballerina.io/learn/tools-ides/intellij-plugin/using-the-intellij-plugin/#creating-a-new-ballerina-file)
+- [**Using the features of the plugin**](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features/)
+   - [Running Ballerina Programs](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#running-ballerina-programs)
+    - [Debugging Ballerina Programs](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#debugging-ballerina-programs) 
+    - [Sequence Diagram View](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-the-sequence-diagram)
+    - [Import Modules On-the-fly](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#importing-modules-on-the-fly)
+    - [Auto Import Unambiguous Modules](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#importing-unambiguous-modules)
+    - [Find Usages](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#finding-usage)
+    - [Go To Definition](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-definitions)
+    - [Formatting](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#formatting-ballerina-codes)
+    - [Parameter Information](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-details-of-parameters)
+    - [Quick Documentation](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#viewing-documentation)
+    - [Annotation Fields Suggestions](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#adding-annotation-fields-via-suggestions)
+    - [File Templates](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#using-file-templates)
+    - [Live Templates and Code snippets](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#using-code-snippet-templates)
+    - [Spell Checking](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#checking-spellings)
+    - [Semantic Analyzing](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#analyzing-semantics)
+    - [Code Folding](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#code-folding) 
+- [**Developer Guide**]((https://ballerina.io/learn/tools-ides/intellij-plugin/#developer-guide))
+    - [Testing/Debugging the plugin using IntelliJ]((https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#testing/debugging-the-plugin-using-intelliJ))
```

Comment:
```suggestion
    - [Testing/Debugging the plugin using IntelliJ](https://ballerina.io/learn/tools-ides/intellij-plugin/using-intellij-plugin-features#testing/debugging-the-plugin-using-intelliJ)
```

Reply:
Removed the additional pair of brackets.